ngraph/CMakeLists.txt

```diff
@@ -490,6 +490,8 @@ if(NGRAPH_JSON_ENABLE)
     include(cmake/external_json.cmake)
 endif()
 
+add_subdirectory(../openvino openvino)
```

Comment:
why do you need to add openvino folder here and use relative path?
CMake on top level already contains include of OpenVINO folder and you can use openvino target to add dedpendency.

Reply:
agree. Adding `add_subdirectory` in root `CMakeLists.txt` is enough. The scope of visibility is propagated for all subdirectoreis

Reply:
During style check step in CI NGraph is building separately. Unfortunately i can't avoid including this file ):

Reply:
Please do not merge with this WA. This issue should be solved on CI side.

Reply:
@azhogov Do we still build nGraph from local folder in the code style check step?

Reply:
The fix was merged https://github.com/openvinotoolkit/openvino/pull/1573, so this WA is not needed anymore as well as `include_guard` in `openvino/CMakeLists.txt`

Reply:
ok